### PR TITLE
Footer project section 1427

### LINKED ISSUE #1427: Program areas: Add join a project section right above the footer
### Overview
Add join a project section right above the footer on the Program Areas page as per figma specs

### Action Items
- [ ] Add join a project section right above the footer (reference the figma design linked below)

### Resources/Instructions
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4452%3A21046
https://www.hackforla.org/program-areas.html

### PR DESCRIPTION
**Pull request by GLRJr**
Fixes #1427  
Added the footer to the project section, but it needs approval from design team because not exactly like figma.

Before I pushed I checked upstream and I had to fix merge conflicts, the CONTRIBUTION documentation for section 7a should to have a step added on what to do after merges are fixed because they are unclear.

Also, do we delete, branches after they are merged?  because in the contribution section doesn't have anything after completing a push.

### CODE REVIEW

**Comment by akibrhast** [comment]
Hey @GLRJr I am getting the following errors, when I checkout your pr and try to run it locally. Could you please delete your `_site` directory and `jekyll-metadata` file and see/fix the issue on your end.

<details>
<summary>Error Message</summary>

```bash
C:\Users\akibr\Desktop\SpringBoardProjects\website>docker compose up
[+] Running 2/2
 - Network website_default  Created                                                        0.0s
 - Container hfla_site      Created                                                        0.8s
Attaching to hfla_site
hfla_site  | ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux-musl]
hfla_site  | Configuration file: _config.yml
hfla_site  | Configuration file: _config.docker.yml
hfla_site  |             Source: .
hfla_site  |        Destination: /srv/jekyll/_site
hfla_site  |  Incremental build: enabled
hfla_site  |       Generating... 
hfla_site  | jekyll 3.9.0 | Error:  Invalid CSS after "}": expected "}", was "" on line 208
hfla_site  |   Conversion error: Jekyll::Converters::Scss encountered an error while converting 
'assets/css/main.scss':
hfla_site  |                     Invalid CSS after "}": expected "}", was "" on line 208        
hfla_site  | /usr/gem/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:123:in `rescue in convert': Invalid CSS after "}": expected "}", was "" on line 208 (Jekyll::Converters::Scss::SyntaxError)
hfla_site  |    from /usr/gem/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:118:in `convert'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:101:in `block in convert'hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `each'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `reduce'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `convert'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:83:in `render_document'  
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:62:in `run'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:479:in `render_regenerated'  
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:472:in `block in render_pages'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `each'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `render_pages'        
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:192:in `render'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:71:in `process'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/command.rb:28:in `process_site'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:65:in `build'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:36:in `process'    
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `block in start'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `each'       
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `start'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'       
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'    
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/exe/jekyll:15:in `<top (required)>'
hfla_site  |    from /usr/gem/bin/jekyll:23:in `load'
hfla_site  |    from /usr/gem/bin/jekyll:23:in `<main>'
hfla_site  | /srv/jekyll/_sass/components/_program-areas.scss:208: Invalid CSS after "}": expected "}", was "" (Sass::SyntaxError)
hfla_site  |    from /srv/jekyll/_sass/main.scss:58
hfla_site  |    from (sass):3
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:1308:in `expected'        
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:1238:in `expected'        
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:1233:in `tok!'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:690:in `block'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:681:in `ruleset'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:707:in `block_child'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:700:in `block_contents'   
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:137:in `stylesheet'       
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/scss/parser.rb:41:in `parse'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/engine.rb:414:in `_to_tree'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/engine.rb:321:in `to_tree'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:323:in `block in visit_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:88:in `block in with_import'    
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:88:in `with_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:322:in `visit_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'  
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block (2 levels) in visit_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `map'    
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:325:in `block in visit_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:88:in `block in with_import'    
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:88:in `with_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:322:in `visit_import'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `block in visit'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:79:in `block in with_base'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:135:in `with_frame'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/stack.rb:79:in `with_base'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:158:in `visit'  
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `block in visit_children'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `map'        
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:52:in `visit_children'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:167:in `block in visit_children'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:179:in `with_environment'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:166:in `visit_children'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `block in visit'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:186:in `visit_root'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/base.rb:36:in `visit'      
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:157:in `visit'  
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/visitors/perform.rb:10:in `visit'   
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/root_node.rb:36:in `css_tree'       
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/tree/root_node.rb:20:in `render'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass/engine.rb:290:in `render'
hfla_site  |    from /usr/gem/gems/sass-3.7.4/lib/sass.rb:57:in `compile'
hfla_site  |    from /usr/gem/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:119:in `convert'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:101:in `block in convert'hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `each'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `reduce'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:99:in `convert'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:83:in `render_document'  
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/renderer.rb:62:in `run'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:479:in `render_regenerated'  
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:472:in `block in render_pages'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `each'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:471:in `render_pages'        
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:192:in `render'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/site.rb:71:in `process'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/command.rb:28:in `process_site'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:65:in `build'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/build.rb:36:in `process'    
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `block in start'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `each'       
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:93:in `start'      
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'       
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'    
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
hfla_site  |    from /usr/gem/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
hfla_site  |    from /usr/gem/gems/jekyll-3.9.0/exe/jekyll:15:in `<top (required)>'
hfla_site  |    from /usr/gem/bin/jekyll:23:in `load'
hfla_site  |    from /usr/gem/bin/jekyll:23:in `<main>'
hfla_site exited with code 1
```

</details>

**Review by akibrhast — CHANGES_REQUESTED** [review]
I am unable to run this locally during review, I have posted the error messages I am receiving. Could you please confirm that the errors are fixed.

**Comment by akibrhast** [comment]
@GLRJr Could you please provide any updates regarding this ?

**Review by akibrhast — DISMISSED** [review]
@GLRJr  , Nice work on this PR so far. These are some notes I have while reviewing this PR

- Please turn of your IDE's auto format features for the website repo, its causing unnecessary line changes in the PRs and making it very difficult to figure out which was the pertinent lines that was really changed.

- Please resolve merge conflicts



>  Added the footer to the project section, but it needs approval from design team because not exactly like figma.

- Could you please describe with images(if possible or necessary )how is it deferring from Figma. @daniellex0 , who is the final design person on this page. Could we tag them onto this conversation ?


> Before I pushed I checked upstream and I had to fix merge conflicts, the CONTRIBUTION documentation for section 7a should to have a step added on what to do after merges are fixed because they are unclear.

- Please open an issue and create a pull request linking that issue and apply the necessary changes where you believe they ought to be and describing why you believe it ought to be there.

> Also, do we delete, branches after they are merged? because in the contribution section doesn't have anything after completing a push.

- It's not necessary, but you can if you want to. Personally I mass delete all my branches every few months.

**Review by akibrhast — CHANGES_REQUESTED** [review]
Requested changes and tagged UI in the above comment

**Comment by akibrhast** [comment]
Moved this to Development meeting board to discuss why it has been sitting in PR for so long, and what can we do about it . Also making this into a draft until requested changes has been applied by @GLRJr . Once the changes are applied, please undraft it.

**Comment by akibrhast** [comment]
@jbubar reached out to @GLRJr on slack, going to touch base tomorrow.

**Comment by GLRJr** [comment]
@jbubar and I worked on this, I have to do little bit of editing margins.
ETA: Tomorrow

**Comment by GLRJr** [comment]
Made changes to margins, I think my browser settings were making things weird. but hopefully you guys think this looks good lol

image

**Comment by akibrhast** [comment]
Based on the figma design I think the foot of the person is supposed to be aligned with the text on the right that does not seem to be the case in this pull request.

<details>
<summary>Figma Design</summary>

image

</details>


<details>
<summary>Your PR</summary>

image


</details>

**Review by akibrhast — DISMISSED** [review]
Based on the figma design I think the foot of the person is supposed to be aligned with the text on the right that does not seem to be the case in this pull request. There also seems to be an unusual amount of space between the two paragraphs on the right.

This was just a preliminary check, I will update this comment with a few more things that caught my eye later tonight

<details>
<summary>Figma Design</summary>

image

</details>


<details>
<summary>Your PR</summary>

image


</details>

**Comment by GLRJr** [comment]
I think we might be looking at two different things because my figma isn't aligned, if you have some time today we can look at this together
image

**Comment by GLRJr** [comment]
I made changes to the margins this should to remove the space in  between the paragraphs and image, how does it look? 
image

**Comment by akibrhast** [comment]
> I think we might be looking at two different things because my figma isn't aligned, if you have some time today we can look at this together

@GLRJr You are right, I was looking at the wrong figma design. I am going to dismiss the review and redo it.

My apologies.

File: pages/program-areas.html [hunk 1]
```diff
@@ -11,8 +11,27 @@ <h1>Program Areas</h1>
             from environmental justice to voter representation, in Los Angeles and far beyond.
         </p>
     </div>
-    <div class="program-areas-header__hero-image">
-        {% include svg/program-areas-hero.svg %}
```
Comment (jbubar) [inline pages/program-areas.html:15]: Was this line supposed to be deleted?

Comment (GLRJr) [reply pages/program-areas.html:15]: is this for me, because I didn't do anything with the header

**Pull request merged**